File: client/src/main/java/com/jraska/github/client/ui/SimpleTextModel.java
package com.jraska.github.client.ui;

import android.widget.TextView;
import com.airbnb.epoxy.EpoxyModel;
import com.jraska.github.client.R;

final class SimpleTextModel extends EpoxyModel<TextView> {
  private final String text;

  public SimpleTextModel(String text) {
    this.text = text;
  }

  @Override protected int getDefaultLayout() {
    return R.layout.item_simple_text;
  }

  @Override public void bind(TextView textView) {
    textView.setText(text);
  }
}

File: feature/ui-common-api/src/main/java/com/jraska/github/client/users/ui/SimpleTextModel.kt
package com.jraska.github.client.users.ui

import android.widget.TextView
import com.airbnb.epoxy.EpoxyModel
import com.jraska.github.client.ui.R

class SimpleTextModel(private val text: String) : EpoxyModel<TextView>() {

  override fun getDefaultLayout(): Int {
    return R.layout.item_simple_text
  }

  override fun bind(textView: TextView) {
    textView.text = text
  }
}
